lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -120,46 +120,17 @@ class HelpPanel extends React.Component {
           </Box>
         </Box>
         <Spacing bottom={10} />
-        <Box centered displayInlineBlock>
+        <Box centered displayInlineBlock fullWidth styledPadding="0 26px">
           {helpItems && (
             <For each="item" index="i" of={helpItems}>
-              <Box displayInlineBlock fullWidth styledHeight={"70px"}>
-                <HelpPanelItem
-                  key={i}
-                  terria={this.props.terria}
-                  viewState={this.props.viewState}
-                  content={item}
-                />
-              </Box>
+              <HelpPanelItem
+                key={i}
+                terria={this.props.terria}
+                viewState={this.props.viewState}
+                content={item}
+              />
             </For>
           )}
-          <Box styledHeight={"64px"}>
-            <Box styledMargin={"auto 25px"}>
-              <Text semiBold extraLarge textDark uppercase>
-                <a
-                  href="https://terriajs.gitbook.io/terriajs/"
-                  target="_blank"
-                  rel="noreferrer noopener"
-                  onClick={() =>
-                    this.props.terria.analytics?.logEvent(
-                      Category.help,
-                      HelpAction.userGuideOpened
-                    )
-                  }
-                >
-                  <BoxSpan displayInlineBlock styledPadding={"0 10px 0 0"}>
-                    {t("helpPanel.terriaMapUserGuide")}
-                  </BoxSpan>
-                  <StyledIcon
-                    glyph={Icon.GLYPHS.externalLink}
-                    displayInline
-                    styledWidth={"15px"}
-                    fillColor={this.props.theme.textDark}
-                  />
-                </a>
-              </Text>
-            </Box>
-          </Box>
```

Comment:
Map user guide link is now generated from `Terria.configParameters.helpItems` just like the other help menu items.

Reply:
Have the user guide link been added to all the other maps?

Reply:
Nope. Have to do that post deploy to avoid an extra link in the help menu. I have noted that in the parent issue.